helm/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.serviceAccount.create -}}
```

Comment:
I"m not sure if we need explicit serviceannount here, if not provided, IIRC kubernetes creates a default service account if not explicitly created.

What would be the use case for that account?

Reply:
It's a leftover from the helm template, I'll remove it.

Reply:
fixed in commit 11b315756d7fd4748bbc1653d89e41ac1a9f5718